DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Reports/ReportsController.cs

```diff
@@ -39,12 +39,21 @@ public IActionResult Index()
             );
 
             var activity = activityService.GetFilteredActivity(centreId, filterData);
+            var filterModel = new ActivityFilterModel
+            {
+                CourseCategoryName = "All categories",
+                CustomisationName = "Customisation",
+                JobGroupName = "Job group",
+                ReportIntervalName = nameof(ReportInterval.Months),
+                StartDate = DateTime.Now.AddYears(-1),
+                EndDate = DateTime.Now
+            };

```

Comment:
Could we make a constructor for this model to build the VM from the filterData object? The values should be "All" if there's no filter for job group, course category and customisation. If customisation is set OR if user is not managing all courses, we should not display course category filter.